gateways/adyen/request_builder.go

```diff
@@ -114,6 +116,24 @@ func buildAuthRequest(authRequest *sleet.AuthorizationRequest, merchantAccount s
 	return request
 }
 
+// addPaymentMethod add payment method to the adyen payment request.
+func addPaymentMethod(authRequest *sleet.AuthorizationRequest, request *checkout.PaymentRequest) {
```

Comment:
Would it make sense to combing this and `addPaymentSpecificFields` into a single method that takes care of all payment related request building logic?

Reply:
👍